doc/OnlineDocs/contributed_packages/mindtpy.rst

```diff
@@ -71,71 +113,185 @@ MindtPy also supports setting options for mip solver and nlp solver.
+
+.. _Kronqvist et al., 2020: https://link.springer.com/article/10.1007/s10107-018-1356-3
+.. _Bernal et al., 2021: http://www.optimization-online.org/DB_HTML/2021/06/8452.html
+
+A usage example for regularized OA is as follows:
 
 .. code::
 
-  >>> import pyomo.environ as pyo
-  >>> model = pyo.ConcreteModel()
+  >>> pyo.SolverFactory('mindtpy').solve(model,
+  ...                                    strategy='OA',
+  ...                                    mip_solver='cplex',
+  ...                                    nlp_solver='ipopt',
+  ...                                    add_regularization='level_L1' 
+  ...                                    # alternative regularizations
+  ...                                    # 'level_L1', 'level_L2', 'level_L_infinity', 
+  ...                                    # 'grad_lag', 'hess_lag', 'hess_only_lag', 'sqp_lag'
+  ...                                    )
+  >>> model.objective.display()
 
-  >>> model.x = pyo.Var(bounds=(1.0, 10.0), initialize=5.0)
-  >>> model.y = pyo.Var(within=Binary)
 
-  >>> model.c1 = Constraint(expr=(model.x-4.0)**2 - model.x <= 50.0*(1-model.y))
-  >>> model.c2 = pyo.Constraint(expr=model.x*log(model.x)+5.0 <= 50.0*(model.y))
-  
-  >>> model.objective = pyo.Objective(expr=model.x, sense=pyo.minimize)
-
-  Solve the model using single tree implementation in MindtPy
-  >>> pyo.SolverFactory('mindtpy').solve(
-  ...    model, strategy='OA',
-  ...    mip_solver='cplex_persistent', nlp_solver='ipopt', single_tree=True)
+Solution Pool Implementation
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+MindtPy supports solution pool of the MILP solver, CPLEX and GUROBI. With the help of the solution, MindtPy can explore several integer combinations in one iteration. 
+
+A usage example for OA with solution pool is as follows:
+
+.. code::
+
+  >>> pyo.SolverFactory('mindtpy').solve(model,
+  ...                                    strategy='OA',
+  ...                                    mip_solver='cplex_peristent',
+  ...                                    nlp_solver='ipopt',
+  ...                                    solution_pool=True,
+  ...                                    num_solution_iteration=10 # default=5
```

Comment:
I think this code snippet is missing a comma after the `num_solution_iteration` option.